### PR TITLE
Fix/private route

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**enhancement : #13**
- Sign up
- Style sign up
- Private routes
- Auth login / logout
- Last page redirect

### CODE REVIEW

File: src/components/login/UserLogin.jsx [hunk 1]
```diff
@@ -15,7 +19,11 @@ const UserLogin = ({ toggleLogin }) => {
     email: "",
     password: "",
   });
-  const { authDispatch } = useAuth();
+  const { setAuth } = useAuth();
+
+  // useEffect(() => {
+  //   auth.isLoggedIn && navigate("/");
+  // }, [auth.isLoggedIn]);
```
Comment (prathmeshjagtap) [inline src/components/login/UserLogin.jsx:26]: remove commented  code

File: src/components/sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -1,53 +1,73 @@
...
+
+        <NavLink
+          to="/videos"
+          style={getActiveStyleBar}
+          className="side-icon-div"
+        >
+          {/* <div className="side-icon-div"> */}
+          <span className="material-icons side-icon">explore</span>
+          <p>Explore</p>
+          {/* </div> */}
```
Comment (prathmeshjagtap) [inline src/components/sidebar/Sidebar.jsx:28]: remove commented code

**Review by prathmeshjagtap — APPROVED** [review]

**Review by ajay-mandaviya — APPROVED** [review]
everything looks good

File: src/components/login/UserLogin.jsx (hunk 1 above)
Comment (tanishk15sharma) [reply src/components/login/UserLogin.jsx:26]: Okay

File: src/components/sidebar/Sidebar.jsx (hunk 2 above)
Comment (tanishk15sharma) [reply src/components/sidebar/Sidebar.jsx:28]: sure

**Pull request merged**